coverage_crawler/crawler.py

```diff
@@ -211,7 +211,7 @@ def run(website, driver):
     return saved_sequence
 
 
-def run_all():
+def run_all(website):
```

Comment:
`run_all` now doesn't apply anymore, maybe rename it to `run` and rename `run` to `run_in_driver`.